libs/expo/betterangels/src/lib/screens/Client/Client.graphql

```diff
@@ -6,7 +6,16 @@ query ClientProfile($id: ID!) {
       dateOfBirth
       gender
       displayGender
+      displayPronouns
```

Comment:
Hi @Davit-BetterAngels 

When the field is empty, the field should not be visible. However, we will create a ticket for this so I can approve this PR.

Reply:
@shiozakiyuko so, only display a field if it has a value? I think that's a small enough change to make here. We don't need a new ticket for that

Reply:
done